tgozo19/laravel-codegen
# Add support for creating model with migration

## Description

This pull request adds support for creating a model when creating a migration from the package. To create the model, the `-m` or `--m` option should be specified in the command for creating the migration.

## Changes

- Added support for the `-m` and `--m` options in the migration creation command

## How to test

To test this feature, run the migration creation command with the `-m` or `--m` option, like so:

```
php artisan codegen:migration --with-fields -m
```

This should create both a migration file and a model file for the `users` table.
